libs/wingc/src/jsify.rs

```diff
@@ -1667,6 +1507,10 @@ fn struct_filename(s: &String) -> String {
 	format!("./{}.Struct.js", s)
 }
 
+fn struct_name_from_filename(s: &String) -> String {
```

Comment:
Do Windows file systems today still have the silly property of being case insensitive? Do we really want to have this dependency between file name and struct name? One alternative would be adding some [id](https://json-schema.org/understanding-json-schema/structuring.html#id) into the schema file which translates into a struct name.

Reply:
Was able to remove this with recent refactor changes.